templates/step/viewimg.html.ep

```diff
@@ -1,23 +1,43 @@
 <div class="aligncenter" data-image="<%= url_for('test_img', filename => $screenshot) %>" id="step_view">
-    Candidate needle:
-    <select id="needlediff_selector">
-        <option data-areas="[]" data-matches="[]">-None-</option>
-        % for my $needle (@$needles) {
-            % my $title = $needle->{min_similarity} . "%: " . $needle->{name};
-            <option data-image="<%= $needle->{image} %>"
-                    data-areas="<%= JSON::to_json($needle->{areas})%>"
-                    % if ($needle->{selected}) {
-                    selected="selected"
-                    % }
-                    data-matches="<%= JSON::to_json($needle->{matches}) %>">
-                <%= $title %></option>
-        % }
-    </select>
+    <span id="candidate_needles">
+        Candidate needle:
+        <select id="needlediff_selector">
+            <option data-areas="[]" data-matches="[]">-None-</option>
+            % for my $needle (@$needles) {
+                % my $title = $needle->{min_similarity} . "%: " . $needle->{name};
+                <option data-image="<%= $needle->{image} %>"
+                        data-areas="<%= JSON::to_json($needle->{areas})%>"
+                        % if ($needle->{selected}) {
+                        selected="selected"
+                        % }
+                        data-matches="<%= JSON::to_json($needle->{matches}) %>">
+                    <%= $title %></option>
+            % }
+        </select>
 
-    % if (is_operator) {
-        %= link_to 'Create new needle' => url_for('edit_step')
-    % }
+        % if (is_operator) {
+            %= link_to 'Create new needle' => url_for('edit_step')
+        % }
+    </span>
+    <span style="float;right;" id="bug_report">
```

Comment:
This does not push the text to the right, it's attached right next to "Create new needle". Ideas?